src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockGoogleServicesDAO.scala

```diff
@@ -106,6 +106,12 @@ class MockGoogleServicesDAO extends GoogleServicesDAO {
 
   override def trialBillingManagerRemoveBillingAccount(projectName: String, targetUserEmail: String): Boolean = false
 
+  override def deleteGoogleGroup(groupEmail: String): Unit = Unit
+  override def checkGoogleGroupExists(groupEmail: String): Boolean = true
+  override def createGoogleGroup(groupName: String): Option[String] = Option("new-google-group@support.something.firecloud.org")
+
+  override def addMemberToAnonymizedGoogleGroup(groupName: String, targetUserEmail: String): Option[String] = Option("user-email@something.com")
```

Comment:
In addition to the tests for the paths that are hit in `UserApiServiceSpec`, I would recommend unit tests for these dao methods as well. `HttpGoogleServicesDAOSpec` seems to make the most sense for them.

Reply:
there isn't currently a way to test the google calls in firecloud-orchestration. however, I added unit tests that ensure that the system still works even if those calls to google fail (in new UserServiceSpec.scala)